api/server/server_linux.go

```diff
@@ -67,6 +67,7 @@ func (s *Server) newServer(proto, addr string) ([]serverCloser, error) {
 	return res, nil
 }
 
+// AcceptConnections allows clients to connect to the API server.
```

Comment:
Here is crucial info, that we're telling daemon that we're ready to accept connections. That's why we need `d` parameter.